src/compiler/transformers/decorators-to-static/convert-decorators.ts

```diff
@@ -72,6 +78,14 @@ const visitClassDeclaration = (
     listenDecoratorsToStatic(diagnostics, decoratedMembers, filteredMethodsAndFields);
   }
 
+  // Handle static members that are initialized
+  const hasStaticMembersWithInit = classMembers.some(hasStaticInitializerInClass);
+  if (hasStaticMembersWithInit) {
+    filteredMethodsAndFields.push(
+      createStaticGetter('stencilHasStaticMembersWithInit', convertValueToLiteral(hasStaticMembersWithInit))
```

Comment:
would it be helpful to declare a constant for this string?

`createStaticGetter` doesn't seem to have any type safety or help to enforce a contract with a known list of static getters — the `propName` param just has type `string`, instead of `"stencilHasStaticMembersWithInit" | "is" | "watchers" | ...`

maybe something to file a ticket about and improve later

Reply:
Yeah, I'm not sure why it isn't like that currently. It definitely doesn't help the searchability of this constant (or any of it's siblings, `is`, `properties`, etc.). I stubbed out STENCIL-856 in JIRA and added a TODO in 3c87ded35